src/js/stores/MesosStateStore.js

```diff
@@ -283,6 +283,13 @@ class MesosStateStore extends GetSetBaseStore {
     });
   }
 
+  getSchedulerTasksMap() {
+    return this.getSchedulerTasks().reduce(
+      (acc, { id, ...rest }) => ({ [id]: rest, ...acc }),
```

Comment:
Is there a special reason why we don't want the task id to be part of the task in the schedulerTasksMap?

Reply:
There's no special reason...  I didn't add the `id` as it's an `id` to task data map and thus the `id` would redundant, but I'm happy to add the `id` if you think that this adds value.

Reply:
Right, you already have the id when calling it, makes sense to me 👍